pkg/csi_driver/node.go

```diff
@@ -277,6 +294,9 @@ func (s *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolu
 	if mounted {
 		// Already mounted
 		klog.V(4).Infof("NodeStageVolume succeeded on volume %v to staging target path %s, mount already exists.", volumeID, stagingTargetPath)
+		if err := s.nodeStageVolumeUpdateLockInfo(ctx, req); err != nil {
+			return nil, status.Errorf(codes.Internal, "failed to store lock info after NodeStageVolume succeeded on volume %v to path %s: %v", volumeID, stagingTargetPath, err.Error())
+		}
 		return &csi.NodeStageVolumeResponse{}, nil
```

Comment:
Log the nodeStageVolume success message here

Reply:
Done